src/pages/home/report/ReportActionCompose.js

```diff
@@ -241,24 +245,61 @@ class ReportActionCompose extends React.Component {
         return this.props.translate('reportActionCompose.writeSomething');
     }
 
+    /**
+     * Set text input value and selection, then call updateComment with newText parameter
+     *
+     * @param {String} newText
+     * @param {Object} selection: {start: number, end: number}
+     */
+    setTextAndSelectionOfTextInput(newText, selection) {
+        if (Platform.OS === 'web') {
+            // react-native-web
+            // because setNativeProps in react-native-web is slow and doesn't have good support
+            this.textInput.value = newText;
+            this.textInput.setSelectionRange(selection.start, selection.end);
+        } else {
+            this.textInput.setNativeProps({text: newText, selection});
+
+            // Relasing selection handling to system
+            setTimeout(() => this.textInput.setNativeProps({selection: {start: undefined, end: undefined}}), 0);
+        }
```

Comment:
Let's discuss this first. I have a couple of questions.
1. why `because setNativeProps in react-native-web is slow and doesn't have good support`? Any reasons to support your comment.
2. Use of `setTimeout`. Why is this needed?

Reply:
It is `setTimeout` 0, you can read more about it here:
https://stackoverflow.com/questions/779379/why-is-settimeoutfn-0-sometimes-useful
For this code it is to ensure that the previous `setNativeProps` is done then do second `setNativeProps`. 

As for `setNativeProps` for react-native-web, please wait.

Reply:
And this is very confusing that you are first settings the selection and then releasing it.

Reply:
If we don't do that, the selection will stay or come back at that position.